### PR TITLE
Fix deprecation warning

### PR DESCRIPTION
**Pull request by felixdivo**
Fixes None.

### Summary

The argument has been [renamed](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html).

### CODE REVIEW

**Comment by madtoinou** [comment]
Hi @felixdivo,

Changing this line breaks backward compatibility with python 3.7 (see #1573). @dennisbader told me that darts will drop python3.7 support soon but in the meantime, we cannot merge this one

**Comment by dennisbader** [comment]
Python 3.7 end of life is actually today, so I guess we can drop support for the next release. But we would have to drop 3.7 from CI before.

**Comment by dennisbader** [comment]
Python3.7 support will be dropped in #1864. Let's wait until it's merged.

**Comment by felixdivo** [comment]
That has happened now :)

File: darts/datasets/__init__.py
Comment (dennisbader) [inline darts/datasets/__init__.py:614]: Actually, can we not just drop the `lineterminator` in general? Then we can avoid the pandas version issues.

Comment (felixdivo) [reply darts/datasets/__init__.py:614]: I guess so but did not try it. If a test cateches it anyways, feel free to apply this suggestion.

```suggestion
            output_df.to_csv(dataset_path)
```

Comment (dennisbader) [reply darts/datasets/__init__.py:614]: I tested locally and it seems to work.

**Review by dennisbader — APPROVED** [review]
LGTM, thanks @felixdivo 💯

**Pull request merged**
